test/Driver/help.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: %swiftc_driver -help | %FileCheck -check-prefix CHECK -check-prefix CHECK-SWIFTC %s
```

Comment:
I suggest adding a comment at the top of the file like: Ensure that the help text from the driver accords with whether it is invoked with swift or swiftc.

Reply:
Hm, why? That seems like the kind of comment that would get out of date with the actual contents of the file, if it ever starts being used to test anything else.